### DISCUSSION TITLE
Instructor Change

### DISCUSSION THREAD

**Discussion by metasean** [USU-ITLS-x265-23-summer/class-repo#159] [Announcements]
Hey Folks,

As some of you know, I've been dealing with medical issues, and unfortunately, I need to step away from this course. 😞 

The good news is that you're new instructor is **_amazing_**.  His name is Ken Snyder.  Besides being a professional developer, he's also (a) a co-founder of UtahJS, (b) has run the SLC UtahJS meetups for over a decade, and (c) spearheads the annual UtahJS conference.  Needless to say, he knows a thing or two about web development, presenting on new topics, and the web development community in general 😉.  While he's never been an instructor of record before, Ken is an absolute rockstar mentor!

Ken will technically take over on July 1st, but you may see him around here before then.  

I'm sorry that I'm not going to be able to see your final projects, but I also know that y'all will be in outstanding hands! 

Also, _don't forget that a third of your grade comes from contributions_ to the [class-repo](https://github.com/USU-ITLS-x265-23-summer/class-repo) and answering your classmates questions here in the class [discussion](https://github.com/orgs/USU-ITLS-x265-23-summer/discussions)!  So, make sure you're actively supporting each other!!!

~sean

**Comment by audreyhki** [comment]
Hope you get well soon! 
Thanks for all your instruction!

**Comment by bst3ck** [comment]
I am so sorry to hear about your recent medical issues. I know it is never easy to deal with those very personal issues and make tough decisions like this. Hopefully, you will be able to spend some time recovering and focusing on getting better. I appreciate all the help you have given this semester to me and I am sure everyone else feels the same way. I wish you the best and a fast recovery. Good luck!!
